kube/src/config/auth.rs

```diff
@@ -256,27 +293,32 @@ mod test {
           name: generic-name
         current-context: generic-name
         kind: Config
-        preferences: {}
+        preferences: {{}}
         users:
         - name: generic-name
           user:
             auth-provider:
               config:
-                cmd-args: '{\"something\": \"else\", \"credential\" : {\"access_token\" : \"my_token\"} }'
+                cmd-args: '{{"something": "else", "credential": {{"access_token": "my_token", "token_expiry": "{expiry}"}}}}'
```

Comment:
is this a breaking change in templating? im struggling to find info on this, but i only found search results with single curly braces

Reply:
No. It's just escaping `{` and `}` in Rust's format string. https://stackoverflow.com/a/25570140

Reply:
It's necessary because I'm replacing `{expiry}` timestamp before parsing. Is there a better way?